tests/Doctrine/Tests/Common/Cache/CouchbaseCacheTest.php

```diff
@@ -7,7 +7,8 @@
 use Doctrine\Common\Cache\CouchbaseCache;
 
 /**
- * @requires extension couchbase
+ * @requires extension couchbase >=1.0
```

Comment:
Will this test run on Travis-CI with the new config (I'm not even sure if it was running before 😂)?

Reply:
you're correct; these tests did not run before, because no version of the extension was installed.

it seemed like a good idea to be very specific in this old test case since the new test case is also very specific.

we might want to run a different build with an older version of the extension, some day. not sure. do you think i should revert this annotation?

Reply:
I'm not sure if this adapter is still relevant.

According to https://pecl.php.net/package-changelog.php?package=couchbase PHP 7 support has only been added on `2.2.0` and `doctrine/cache` now requires PHP 7.1 (so that excludes couchbase `v1`).

I've added https://github.com/doctrine/cache/issues/234 to discuss things like this.

Reply:
excellent!